### ISSUE TITLE
Object3DPopulationColocalisation bug?

### ISSUE THREAD

**Issue by tischi** [mcib3d/mcib3d-core#11]
@mcib3d @ssgpers

I am not sure but I think there might be a bug at those three places:

https://github.com/mcib3d/mcib3d-core/blob/master/src/main/java/mcib3d/geom/Objects3DPopulationColocalisation.java#L124

https://github.com/mcib3d/mcib3d-core/blob/master/src/main/java/mcib3d/geom/Objects3DPopulationColocalisation.java#L138

https://github.com/mcib3d/mcib3d-core/blob/master/src/main/java/mcib3d/geom/Objects3DPopulationColocalisation.java#L151

The issues is that these functions can be called without the coloc array being build.
Basically, I think these three functions are missing this statement:

```
if(needToComputeColoc) computeColocalisation();
```

Is that right?

**Comment by mcib3d** [comment]
Hi @tischi ,

Yes I think you are right :) , thanks a lot. Fixed in c2f962c1767e137bbcb8410e1282f32efba36dc2.
